README.md

```diff
@@ -219,19 +212,36 @@ If you wish to transfer command to another starship, simply press '\[' to switch
 **Compiling On Linux**
 -------------------------------------
 
-use the vsbuild.sh script in the root directory.
+1. Install the dependencies. Something like this:
+
+```bash
+sudo apt-get -y install cmake g++ python-dev libboost-python-dev libgl1-mesa-glx freeglut3-dev libopenal-dev libsdl-gfx1.2-dev libvorbis-dev libjpeg-dev libpng-dev libgtk2.0-dev
+```
```

Comment:
Commands should be refixed with a `$` for userland to distinguish each command-line entry.

Reply:
Won't that interfere with copy-pasting?

Reply:
@BenjamenMeyer See my response above. Won't including the `$` interfere with individuals copy-pasting the commands into their terminal? It does in my experience.

Perhaps it would be better to use line continuation characters `\` to break up each of these lengthy `apt` commands?

Reply:
The latter is what I ended up doing.